app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -62,10 +61,7 @@
         \n-&#032;google &lt;query&gt;
         \n-&#032;wikipedia &lt;query&gt;
         \n-&#032;yahoo &lt;query&gt;
-        \n
```

Comment:
Sorry, I added these 3 lines after first adding ping command. Can you include there line?

Reply:
@nhirokinet Should be okay now. Last edit was based on https://github.com/nhirokinet/bluelineconsole/compare/nhirokinet:7c48486...nhirokinet:6a97e1d